### DISCUSSION TITLE
start fe error:catch an exception when setup bdb environment. will exit.

### DISCUSSION THREAD

**Discussion by wz2326** [apache/doris#17180] [A - General / Q&A]
start fe error:catch an exception when setup bdb environment. will exit.
operating environment:chroot
doris version:1.1.3
![image](https://user-images.githubusercontent.com/8318913/221497006-7cb0caef-e867-435e-9473-0b4f2d8ebbde.png)

**Comment by LemonLiTree** [comment]
换1.1.5试试？
